core/src/main/java/software/amazon/awssdk/core/SdkSystemSetting.java

```diff
@@ -75,12 +75,6 @@
      */
     AWS_REGION("aws.region", null),
 
-    /**
-     * Whether the default configuration applied to AWS clients should be optimized for services within the same region.
-     * This will usually include lower request timeouts because requests do not need to travel outside of the AWS network.
-     */
-    AWS_IN_REGION_OPTIMIZATION_ENABLED("aws.inRegionOptimizationEnabled", "false"),
```

Comment:
Are we removing this because it is unused?

Reply:
yes